src/main/java/healthy/lifestyle/backend/user/service/UserServiceImpl.java

```diff
@@ -206,4 +207,14 @@ public void deleteWorkoutFromUser(User user, Workout workout) {
         if (user.getWorkouts() != null) user.getWorkouts().remove(workout);
         userRepository.save(user);
     }
+
+    @Override
+    @Transactional(propagation = Propagation.MANDATORY)
+    public void deleteMentalFromUser(long userId, Mental mental) {
+        User user = userRepository
+                .findById(userId)
+                .orElseThrow(() -> new ApiException(ErrorMessage.USER_NOT_FOUND, userId, HttpStatus.NOT_FOUND));
+        if (user.getMentals() != null) user.getMentals().remove(mental);
```

Comment:
How do you think, should we check whether this particular user has this particular mental entity? (user-mental mismatch)